### ISSUE TITLE
Convert chroot_setup to Python

### ISSUE THREAD

**Issue by kadler** [IBM/ibmichroot#70]
This is a complicated enough that it should be written in a higher level language.

**Comment by markdirish** [comment]
@abmusse are you trying to work on this? I have a local branch with most of it rewritten already, just needs testing for the various edge cases

**Comment by abmusse** [comment]
Yeah I was planning on working on this but have not started on it yet. If you already have most of it written already I can help test things out.
